### ISSUE TITLE
Improve UI/UX

### ISSUE THREAD

**Issue by nehagoel2** [Intelehealth/Android-Mobile-Client#470]
Multiple fixes to improve the app's usability. Prajwal will be working with Vibha to deliver this.
Estimated time to completion - 1 week.

**Comment by vibhabhirud** [comment]
Some feedback from the field and feature requests from the clients:
470.1- Age should be an input rather than a scroll
470.2- Radio button for selection of Yes/No kind of questions
470.3 App directly takes you to the missing compulsory input field
470.4- Questions should appear on screen even after selecting for a drop down. Currently, one has to scroll up for the question 
470.5- Non-general exams should be allowed to make compulsory
470.6- UI of the feedback screen needs to be improvised. eg. skip button could be smaller and the below submit button
470.7- Too many clicks for some input text attributes. Eg.while entering BP in physical exams
470.8- Renaming of additional document pictures
470.9- Ability to attach pictures from gallery or pdf from documents
470.10- Summary of Patient history and family history for further visits

**Comment by nehagoel2** [comment]
470.12- Improve look and feel of Patient Registration Screen

**Comment by vibhabhirud** [comment]
Splash Screen:
SS1- Icon shape. Rounding the edges
SS2- Icon name 'Intelehealth'
SS3- Animation while launching the app

Login Screen
LS1- Show password
LS2- Get inspired by twitter's login
LS-3 Icon for username and password (Get inspired by Motion)
LS4- Device ID should not be seen
LS5- Shading to the background

Home Screen
HS1- System back=App collapses
HS2- Tile look to card view
HS3- Space between icons and text
HS4- Icon change for today's patient and video sync
HS-5 Font weight should be increased in 3 dots menu
HS-6 Icons for all buttons

Patient Screen
PS1- System back and back arrow with a pop-up 'Are you sure you want to go back'
PS2- Autoscroll to the mandatory field
PS3- All mandatory fields should be shown at once
PS4- Add camera in the person image at image view. 
PS4.1-Remove pop-up.
PS4.2- Use system camera instead of App camera
PS4.3- Accept/Decline image
PS4.4- enable the system back at the camera
#PS5- Country, State, Econ Status, Caste, Education- Input+ drop down with autoselect
#PS6- 
#PS7- Input field for Age instead of scroll
#PS8- Limit phone no to max. 10 digits
#PS9- Default selection of gender (female)
PS10- Icons for all each field
PS11- 3 card each for personal data, address and other details
PS12- Move phone number below address

**Comment by prajwalhacker** [comment]
SplashScreen work is completed ! Looking forward to the Login Screen !

**Comment by prajwalhacker** [comment]
Login Screen work is been completed !!! Next - Home Screen !

**Comment by prajwalhacker** [comment]
Home Screen Done !

**Comment by nehagoel2** [comment]
On the login screen can you add a link that says “Can’t log in?”
Like this login screen for Instagram has a link for “Forgot Password”:
https://2.bp.blogspot.com/-ThSCWcwtRV0/WQmIX-w-YmI/AAAAAAAAEmc/m5z0HvOpltAQP2WaJbpoAX-GoIVNVYSaQCPcB/s1600/Screenshot_2017-05-02-10-58-36-94.png

When the person clicks on the “Can’t log in?” link, there should be a pop up message that says “Please contact your System Administrator to reset your password or send an email to support@intelehealth.io”.

Basically need to replicate the functionality that's on the OpenMRS side on the app.
<img width="250" alt="screen shot 2018-06-16 at 11 37 54 am" src="https://user-images.githubusercontent.com/14187848/41496305-278578c0-715a-11e8-925f-66474563df8b.png">

**Comment by prajwalhacker** [comment]
Login screen changes done !!

**Comment by prajwalhacker** [comment]
PS-5 and PS-7 are not been done because the coding might affect the database section !
Other all changes has been done....
Camera and Mandatory fields will be working on by tomorrow.
Thank you !
